cloudnet-wrapper/src/main/java/de/dytanic/cloudnetwrapper/bootstrap/CloudBootstrap.java

```diff
@@ -27,6 +27,13 @@
 
     public static void main(String[] args) throws Exception
     {
+        
+        if (Float.parseFloat(System.getProperty("java.class.version")) < 52D) 
+        { 
+            System.out.println("This application needs at least Java 8 or 10.0.1!"); 
```

Comment:
```suggestion
            System.out.println("This application needs at least Java 8!"); 
```

This is enough, Java 10 is obsolete.